docset/winserver2022-ps/netwnv/Get-NetVirtualizationProviderAddress.md

```diff
@@ -23,48 +23,55 @@ Get-NetVirtualizationProviderAddress [-ProviderAddress <String[]>] [-InterfaceIn
 ```
 
 ## DESCRIPTION
-The **Get-NetVirtualizationProviderAddress** cmdlet gets Provider Addresses configured in Microsoft® Hyper-V® Server 2016 Network Virtualization.
-A Provider Address is an IPv4 or IPv6 address that Network Virtualization uses for multiple virtual Customer Addresses.
-For more information, see [Network Virtualization technical details](https://technet.microsoft.com/library/jj134174.aspx) on TechNet.
 
-You can specify which Provider Addresses to get by using address state, interface index, prefix length, IP address, or VLAN ID.
-You can use this cmdlet to get Provider Addresses for other cmdlets, such as the Set-NetVirtualizationProviderAddress cmdlet or the Remove-NetVirtualizationProviderAddress cmdlet.
+The `Get-NetVirtualizationProviderAddress` cmdlet gets Provider Addresses configured in Microsoft®
+Hyper-V® Server 2016 Network Virtualization. A Provider Address is an IPv4 or IPv6 address that
+Network Virtualization uses for multiple virtual Customer Addresses. For more information, see
+[Network Virtualization technical details](https://technet.microsoft.com/library/jj134174.aspx) on
+TechNet.
+
+You can specify which Provider Addresses to get by using address state, interface index, prefix
+length, IP address, or VLAN ID. You can use this cmdlet to get Provider Addresses for other cmdlets,
+such as the `Set-NetVirtualizationProviderAddress` cmdlet or the
+`Remove-NetVirtualizationProviderAddress` cmdlet.
 
 ## EXAMPLES
 
 ### Example 1: Get Provider Addresses for an interface
-```
-PS C:\> Get-NetVirtualizationProviderAddress -InterfaceIndex 13
+
+```powershell
+Get-NetVirtualizationProviderAddress -InterfaceIndex 13
 ```
 
 This command gets the Provider Address for the interface with the specified index.
 
 ### Example 2: Get a specific Provider Address
-```
-PS C:\>Get-NetVirtualizationProviderAddress -ProviderAddress "172.16.2.3"
+
+```powershell
+Get-NetVirtualizationProviderAddress -ProviderAddress '172.16.2.3'
 ```
 
 This command gets the Provider Address that has the IP address 172.16.2.3.
```

Comment:
We want to backtick this value because it's specific to the command line in the example.

```suggestion
This command gets the Provider Address that has the IP address `172.16.2.3`.
```